catalyst/data/sampler.py

```diff
@@ -154,4 +159,22 @@ def __len__(self) -> int:
         return self.mini_epoch_len
 
 
-__all__ = ["BalanceClassSampler", "MiniEpochSampler"]
+class DistributedSamplerOverSampler(DistributedSampler):
+    def __init__(self, sampler, num_replicas=None, rank=None, shuffle=True):
```

Comment:
and add some minimal readme docs for `DistributedSamplerOverSampler` and `DatasetFromSampler`
like, what are the usecases for their usage?

Reply:
done